CHANGELOG.md

```diff
@@ -6,14 +6,7 @@
 
 ## v0.4.0
 
-### 🛑 Breaking changes 🛑
```

Comment:
Usually we should keep the changelog, the version is still available and if someone was already using it they can keep doing so if they want. In this instance I don't think anyone is using it so I will leave whether to bring the changelog back up to you :)